crypto-common/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "crypto-common"
 description = "Common cryptographic traits"
-version = "0.1.6"
+version = "0.2.0"
```

Comment:
FWIW, if this is starting a new release cycle it would be nice to have more time to do more extensive breaking changes, like experiment with a migration to `hybrid-array`.

Perhaps you could bump the version to a prerelease instead?